br/pkg/lightning/config/global.go

```diff
@@ -168,6 +185,8 @@ func LoadGlobalConfig(args []string, extraFlags func(*flag.FlagSet)) (*GlobalCon
 	statusAddr := fs.String("status-addr", "", "the Lightning server address")
 	serverMode := fs.Bool("server-mode", false, "start Lightning in server mode, wait for multiple tasks instead of starting immediately")
 
+	checkOnlyCfg := fs.String("check-only", "", "do check only, supported value: --check-only=default or --check-only=rate,rows")
```

Comment:
@kennytm what do you think about this. I feel this kind of compaction multi config into one field is not very user-friendly.

Reply:
@glorv we could use [`flagext.StringsVar`](https://pkg.go.dev/github.com/carlmjohnson/flagext#StringsVar) here but it only supports the form `--check-only=rate --check-only=rows`